utilities/after_success.sh

```diff
@@ -32,7 +32,7 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
 
         cd ..
         utilities/update_docs_version.sh # Update version in READMEs
-        mvn deploy -DskipTests=true --settings target/travis/settings.xml -P sign-deploy
+        mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy
     else
         mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
```

Comment:
actually, we probably want to leave it outside, so we can see the change in Corvallis after every commit. Regardless, fine with no re-run tests for snapshots (fixing a bad snapshot is simple).